client/src/main/java/com/kritsit/casetracker/client/domain/model/Day.java

```diff
@@ -71,10 +71,13 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        String result = getNumber() + "\n";
+        StringBuilder result = new StringBuilder();
```

Comment:
"Method concatenates strings using + in a loop"
Java strings are immutable so every time they are appended using + they get converted to a StringBuilder and get converted back. Stringbuilder is much better performance.